### PR TITLE
Run tests on GHA.

### PR DESCRIPTION
**Pull request by icemac**

### CODE REVIEW

**Comment by icemac** [comment]
CI seems to be broken because of https://github.com/actions/setup-python/issues/171.

**Comment by icemac** [comment]
Appveyor seems to need a 3.9 wheel of `zope.proxy` for window.

**Comment by icemac** [comment]
Using an older actions/setup-python to get GHA running again.

File: .github/workflows/tests.yml [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+        - ["pypy3", "pypy3"]
+        - ["3.8",   "docs"]
+        - ["3.8",   "coverage"]
+
+    runs-on: ubuntu-latest
+    name: ${{ matrix.config[1] }}
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python
+      uses: actions/setup-python@v2.1.4
```
Comment (mgedmin) [inline .github/workflows/tests.yml:35]: The fix for which this is a workaround has been fixed, so we can go back to 

```suggestion
      uses: actions/setup-python@v2
```

File: tox.ini [hunk 2]
```diff
@@ -1,29 +1,73 @@
...
+extras =
+    test
 
-[testenv:coverage]
-usedevelop = true
-basepython =
-    python3.7
-commands =
-    coverage run -m zope.testrunner --test-path=src []
-    coverage report --fail-under=100
```
Comment (mgedmin) [inline tox.ini:16]: We had 100%, and now we have 99%?  What happened?

**Review by mgedmin — COMMENTED** [review]
Mostly LGTM, some slight comments.

**Comment by mgedmin** [comment]
> Appveyor seems to need a 3.9 wheel of zope.proxy for windows.

Appveyor has a C compiler and shouldn't need any wheels.

The error

>       error: invalid command 'bdist_wheel'

seems to indicate that our appevyor.yml is missing a `pip install -U wheel`?

File: tox.ini (hunk 2 above)
Comment (mgedmin) [reply tox.ini:16]: It's mostly branch coverage (which we had disabled before), but also now line 109 in src/zope/sendmail/maildir.py is no longer covered.

https://github.com/zopefoundation/zope.sendmail/blob/5e3f7868607f9e52f7d029c152b9bc18ce4b6474/src/zope/sendmail/maildir.py#L109

This appears to be a side effect of switching from basepython = python3.7 to basepython = python3 (which is 3.8 on my Ubuntu 20.10)!

Anyway, this is fine, we can climb back to 100% later.

**Comment by mgedmin** [comment]
https://github.com/zopefoundation/meta/pull/54 adds wheel to the appveyor.yml template.

**Comment by sallner** [comment]
@mgedmin So it seems, with the exception of pypy we are all set here. This should be fixed with #38 as far as I understand. Should we merge it now, so we we can process with #38 which and other PRs?

**Comment by mgedmin** [comment]
Yes, let's.

**Pull request merged**